vmdb/spec/lib/workers/schedule_worker_spec.rb

```diff
@@ -566,6 +566,7 @@
       VMDB::Config.any_instance.stub(:config).and_return(
         :ems_refresh => {
           :refresh_interval => 24.hours,
+          :kubernetes       => {:refresh_interval => 15.minutes},
```

Comment:
If you're adding a config value, can you add a test for it below?  Technically, this is not even needed because the use case (of override) is covered by the scvmm case below.